lib/services/sqs_queue.rb

```diff
@@ -41,7 +41,13 @@ def notify_sqs(aws_access_key, aws_secret_key, payload)
     else
         queue = sqs.queues.named(queue_name)
     end
-    queue.send_message(clean_for_json(payload))
+    sqs.send_message(
+      queue_url: queue.url,
+      message_body: clean_for_json(payload),
+      message_attributes: {
```

Comment:
It's very possible that I'm wrong, but I'm not sure that [`message_attributes`](https://github.com/aws/aws-sdk-ruby/blob/1.27.0/lib/aws/sqs/queue.rb#L87-L115) is supported in [version 1.27.0](https://github.com/github/github-services/blob/c7ebee32771af3faed36500dfe239b1605478524/github-services.gemspec#L61) of aws-sdk-ruby.